module.json

```
@@ -16,10 +16,10 @@
       "discord": "Cattegy#7436"
     }
   ],
-  "version": "3.0.0",
+  "version": "3.0.1",
   "compatibility": {
     "minimum": "11",
```

```suggestion
    "minimum": "12",
```

I think min should also be 12? as this fixes problems caused by the API change from 11 to 12

---

As far as I know, the merge of `DocumentData` into `Document` has been introduced as early as v10. It's just the final removal of `data` that has been part of v12.

See here for more details: https://foundryvtt.com/article/v10-data-model/